tasks/disk_settings.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Get list of block devices to test
+  shell: ls /sys/block/ | egrep '^(([shv]|xv)d[a-z])|(nvme[0-9a-z]+)$'
+  register: block_devs
+  changed_when: no
+  check_mode: no
+
+- name: Get block device current disk scheduler
+  shell: cat /sys/block/{{ item }}/queue/scheduler | egrep -q '\[{{ disk_scheduler }}\]'
+  with_items: "{{ block_devs.stdout_lines }}"
+  register: scheduler_test
+  changed_when: scheduler_test.rc == 1
+  failed_when: scheduler_test.rc == 2
+
+- name: Set block device disk scheduler
+  shell: echo {{ disk_scheduler }} > /sys/block/{{ item }}/queue/scheduler
+  with_items: "{{ block_devs.stdout_lines }}"
+  when: scheduler_test.changed
+
+- name: Get block device current readahead setting
+  shell: cat /sys/block/{{ item }}/queue/read_ahead_kb | egrep -q '\[{{ disk_read_ahead_kb }}\]'
```

Comment:
This one should be:

`shell: cat /sys/block/{{ item }}/queue/read_ahead_kb | egrep -q '{{ disk_read_ahead_kb }}'`

Reply:
Oh yes you're right!

However this PR is blocked since we can't test it because of Travis timeout.
And I don't have anymore write permissions on the [Talend/ansible-role-mongodb](https://github.com/Talend/ansible-role-mongodb/tree/system_requirements) forked repo.